sql-statements/sql-statement-show-table-next-row-id.md

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
Please add this file link `/sql-statements/sql-statement-show-table-next-row-id.md` to TOC.md